spec/rails_helper.rb

```diff
@@ -0,0 +1,24 @@
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../config/environment', __dir__)
+
+abort('The Rails environment is running in production mode!') if Rails.env.production?
```

Comment:
maybe we can change line 1 to not be `||=` but just `=` so we always run on test mode, thoughts?

Reply:
mmm, that configuration is to allow you to override the envorionment for when running tests. Why would you remove the possibilty to run a command like `RAILS_ENV=development rpsec` ?

Reply:
hmm that can be weird because you would be deleting and altering your development db. 
Also, development might have the env vars necessary to do http requests to third party services, whereas in test environment they shouldn't be needed since web requests should be blocked. 
I never had to do it but if you find it useful then leave it as it is

Reply:
I agree that it's a weird use case. 

What you are pointing of the dev environment not having the necessary test env variables values makes sense, however I think that if you are running tests on dev mode, there is a reason to do that which may take you to set those env vars. 

Regardless of those points what you are suggesting is to block the possibility (change the default rspec configuration), the only way to run test on a different environment than tests is to manually set te env var before running the tests command like this: `RAILS_ENV=development rspec` , in that sense this line doesn't add any risk. So I don't see why would it make sense to change it